featurebyte/common/validator.py

```diff
@@ -145,3 +147,25 @@ def version_validator(cls: Any, value: Any) -> Any:
     if isinstance(value, str):
         return convert_version_string_to_dict(value)
     return value
+
+
+def duration_string_validator(cls: Any, value: Any) -> Any:
+    """
```

Comment:
Should we reused the logic here: https://github.com/featurebyte/featurebyte/blob/main/featurebyte/common/model_util.py#L15?

Reply:
done